pkg/engine/funcs.go

```diff
@@ -88,6 +90,19 @@ func toYAML(v interface{}) string {
 	return strings.TrimSuffix(string(data), "\n")
 }
 
+func toYAMLPretty(v interface{}) string {
+	var data bytes.Buffer
+	encoder := goYaml.NewEncoder(&data)
```

Comment:
Is it possible to use the existing `sigs.k8s.io/yaml` ? I know `sigs.k8s.io/yaml` is a wrapper around `gopkg.in/yaml.v3`, and probably for encoding yaml simply a passthrough to `gopkg.in/yaml.v3`. But to keep things "consistent" if possible.